pandas/core/window.py

```diff
@@ -312,6 +312,9 @@ class Window(_Window):
     * ``gaussian`` (needs std)
     * ``general_gaussian`` (needs power, width)
     * ``slepian`` (needs width).
+
+    To learn more about the frequency strings, please see `this link
+    <http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases>`__.
```

Comment:
to clarify, the `freq` keyword can accept those, but this keyword is deprecated, so I think it is indeed not needed to add it here